PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskRecoilJetYield.h

```diff
@@ -145,6 +146,9 @@ class AliAnalysisTaskRecoilJetYield : public AliAnalysisTaskEmcalJet {
   TString                             fRhoName;
   AliRhoParameter                     *fRhoParam;
   Bool_t                              fSubMatching;
+  Bool_t                              bMinSubjetPt;
```

Comment:
You should update your ClassDef, and you should really be setting whether members are streaming or not (but perhaps you can get away with this until after QM)